recipes/defaults.rb

```diff
@@ -251,3 +251,30 @@
     hard_restart
   end
 end
+
+namespace :deploy_docker do
+  # These tasks are specific to a docker deployment
+  desc "Deploy the application as a docker image"
+  task :default do
+    pull
+    tag_to_current
+    restart
+  end
+
+  desc "Pull the docker image using a specific tag"
+  task :pull do
+    run "sudo docker image pull #{application}:#{branch}"
```

Comment:
I think this is correct as branch as defined [here](https://github.com/alphagov/govuk-app-deployment/blob/master/recipes/defaults.rb#L5)